lib/src/main/java/net/mossol/bot/model/LocationInfo.java

```diff
@@ -1,9 +1,13 @@
 package net.mossol.bot.model;
 
+import org.springframework.data.annotation.Id;
+
 import lombok.Data;
 
 @Data
```

Comment:
fields can be modified? how about @Value.

Reply:
We have create new `LocationInfo` model class such as https://github.com/AmosDoan/mossol-bot-lib/pull/95/files#diff-59225ed4fbd0ed57e1898891635ad01eR66.
But in this case, we don't know id field issued by MongoDB. If we set @value in this case, `id` field should be set when the class is initialized.